tasks/scheduler_test.go

```diff
@@ -176,7 +168,7 @@ func TestProcessAll(t *testing.T) {
 		{
 			name:      "Test every tx accesses same key",
 			workers:   50,
-			runs:      1,
+			runs:      5,
```

Comment:
did we also want to make an assertion for this test that maxIteration is like 6 or something?